chartfx-math/src/main/java/de/gsi/math/DataSetMath.java

```diff
@@ -850,45 +849,45 @@ public static DataSet mathFunction(final DataSet function1, final DataSet functi
             switch (op) {
             case ADD:
                 newY = Y1 + Y2;
-                nEYN = Math.hypot(eyn1, eyn2);
-                nEYP = Math.hypot(eyp1, eyp2);
+                nEYN = java.lang.Math.hypot(eyn1, eyn2);
+                nEYP = java.lang.Math.hypot(eyp1, eyp2);
                 break;
             case SUBTRACT:
                 newY = Y1 - Y2;
-                nEYN = Math.hypot(eyn1, eyn2);
-                nEYP = Math.hypot(eyp1, eyp2);
+                nEYN = java.lang.Math.hypot(eyn1, eyn2);
+                nEYP = java.lang.Math.hypot(eyp1, eyp2);
                 break;
             case MULTIPLY:
                 newY = Y1 * Y2;
-                nEYN = Math.hypot(Y2 * eyn1, Y1 * eyn2);
-                nEYP = Math.hypot(Y2 * eyp1, Y1 * eyp2);
+                nEYN = java.lang.Math.hypot(Y2 * eyn1, Y1 * eyn2);
+                nEYP = java.lang.Math.hypot(Y2 * eyp1, Y1 * eyp2);
                 break;
             case DIVIDE:
                 newY = Y1 / Y2;
-                nEYN = Math.hypot(eyn1 / Y2, newY * eyn2 / Y2);
-                nEYP = Math.hypot(eyp1 / Y2, newY * eyp2 / Y2);
+                nEYN = java.lang.Math.hypot(eyn1 / Y2, newY * eyn2 / Y2);
+                nEYP = java.lang.Math.hypot(eyp1 / Y2, newY * eyp2 / Y2);
                 break;
             case SQR:
-                newY = TMathConstants.Sqr(Y1 + Y2);
-                nEYN = 2 * Math.abs(Y1 + Y2) * Math.hypot(eyn1, eyn2);
-                nEYP = 2 * Math.abs(Y1 + Y2) * Math.hypot(eyp1, eyp2);
+                newY = MathBase.sqr(Y1 + Y2);
+                nEYN = 2 * java.lang.Math.abs(Y1 + Y2) * java.lang.Math.hypot(eyn1, eyn2);
+                nEYP = 2 * java.lang.Math.abs(Y1 + Y2) * java.lang.Math.hypot(eyp1, eyp2);
                 break;
             case SQRT:
-                newY = TMathConstants.Sqrt(Y1 + Y2);
-                nEYN = Math.sqrt(Math.abs(Y1 + Y2)) * Math.hypot(eyn1, eyn2);
-                nEYP = Math.sqrt(Math.abs(Y1 + Y2)) * Math.hypot(eyp1, eyp2);
+                newY = MathBase.sqrt(Y1 + Y2);
+                nEYN = java.lang.Math.sqrt(java.lang.Math.abs(Y1 + Y2)) * java.lang.Math.hypot(eyn1, eyn2);
+                nEYP = java.lang.Math.sqrt(java.lang.Math.abs(Y1 + Y2)) * java.lang.Math.hypot(eyp1, eyp2);
                 break;
             case LOG10:
-                norm = 1.0 / Math.log(10);
-                newY = TMathConstants.Log10(Y1 + Y2);
-                nEYN = Y1 + Y2 > 0 ? norm / Math.abs(Y1 + Y2) * Math.hypot(eyn1, eyn2) : Double.NaN;
-                nEYP = Y1 + Y2 > 0 ? norm / Math.abs(Y1 + Y2) * Math.hypot(eyp1, eyp2) : Double.NaN;
+                norm = 1.0 / java.lang.Math.log(10);
```

Comment:
Codacy found an issue: [Unnecessary use of fully qualified name 'java.lang.Math.log' due to existing implicit import 'java.lang.*'](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5780327)